packages/react-widgets/src/layers/useEventManager.js

```diff
@@ -0,0 +1,42 @@
+import { EventManager } from 'mjolnir.js';
+import { useEffect, useMemo } from 'react';
+import { useSelector } from 'react-redux';
+import { BASEMAPS } from '@carto/react-basemaps/';
+import { FEATURE_SELECTION_MODES } from '@carto/react-core/';
+
+// When we're using Google as a base map, Nebula layer doesn't work becase Nebula uses the Deck Gl eventManager
+// so in this case we're defining a new EventManager with the Google maps div. On the other hand, in Builder
+// we don't use the c4r GoogleMap component so we need to send a EventManager as parameter
+export default function useEventManager({
+  eventManager,
+  isEdit,
+  isSelected,
+  selectedMode
+}) {
+  const isGmaps = useSelector((state) => BASEMAPS[state.carto.basemap].type === 'gmaps');
```

Comment:
Is not 'gmaps' in any constant? Should we move it to a new one?

Reply:
We don't have a constant for 'gmaps'. Move it to a new one it's not as easy as it sounds because the basemap package also uses that string so probably we would need to add it to the core package and do a refactor. @VictorVelarde maybe we could do it in another PR